release_notes/41-release-notes.adoc

```diff
@@ -425,6 +426,15 @@ The updated image fixes this issue.
 * Fixed an issue where upgrading from RHACS 3.74.3 to RHACS 4.1.1 failed
 with an "insert...violates foreign key restraint" error.
 
+[id="resolved-in-version-413_{context}"]
+=== Resolved in version 4.1.3
+
+Release date: 16 August 2023
+
+* Fixed an issue where {product-title-short} reported an incorrect component version for Tomcat used in an application.
+* Removed a check that prevented a rollback to an earlier {product-title-short} after migration.
+* {product-title-short} now includes `ALL` as a valid value for drop capabilities during policy deployment. The policy implementation has been changed so that if the policy criteria specifies that a deployment must drop a capability (for example, A or B), and a deployment manifest contains `DROP ALL`, it does not violate the policy.
```

Comment:
Remove "during policy deployment".  Rationale: Overloaded term deployment, potentially causing confusion. Also incorrect since the value is available immediately on upgrade.